modules/hackathon-2023/Hackathon2023TopBanner.tsx

```diff
@@ -0,0 +1,86 @@
+import { Grid, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { ReactElement } from 'react';
+
+import { darken } from '../../styles/helpers/color';
+
+const useStyles = makeStyles(
+  (theme) => ({
+    root: {
+      background: `linear-gradient(to bottom, ${darken(theme.palette.primary.dark, 0.3)}, ${theme.palette.primary.dark})`,
+      color: theme.palette.text.secondary,
+      paddingTop: theme.spacing(12),
+    },
+    gridContainer: {
+      padding: theme.spacing(4),
+    },
+    typography: {
+      textTransform: 'uppercase',
+      fontSize: '1em',
+      fontWeight: theme.typography.fontWeightBold,
+    },
+    gridItemLeft: {
+      textAlign: 'center',
+      [theme.breakpoints.up('md')]: {
+        padding: `0 ${theme.spacing(3)}px`,
+        textAlign: 'right',
+        position: 'relative',
+        '&:after': {
+          content: '""',
+          display: 'block',
+          width: '0.1em',
+          top: 0,
+          bottom: 0,
+          background: theme.palette.text.secondary,
+          opacity: 0.5,
+          position: 'absolute',
+          right: 0,
+        },
+      },
+    },
+    gridItemRight: {
+      textAlign: 'center',
+      [theme.breakpoints.up('md')]: {
+        padding: `0 ${theme.spacing(3)}px`,
+        textAlign: 'left',
+      },
+    },
+  })
+);
+
+const Hackathon2023TopBanner = (): ReactElement => {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Grid
+        container
+        className={classes.gridContainer}
+      >
+        <Grid
+          item
```

Comment:
In defining my Grid components, I was forgetting to include the `item` prop, think that was my problem 🙈

Reply:
To be fair we're using an outdated version of MUI. I tried to update it but it exploded in my face immediately - a task for another day! lol